tests/test_data/duckdb_data/expected_export/core/core__count_allergyintolerance_month.cube.csv

```diff
@@ -1,3 +1,3 @@
-"cnt","category","recordedDate_month","code_display","reaction_manifestation_display"
+cnt,category,recordedDate_month,code_display,reaction_manifestation_display
 17,,,,
-16,,"2018-08-01",,
+16,,2018-08-01,,
```

Comment:
I know that Vlad's dashboard sometimes makes assumptions about the quoting scheme in use. Have you tested whether this scheme works correctly? You might have to add some flags to the csv exporter to keep the same quoting behavior if not.

Reply:
Confirmed this format works.